Samples/BackgroundBlur/DX11VideoRenderer/Activate.cpp

```diff
@@ -0,0 +1,214 @@
+#include "pch.h"
```

Comment:
So it seems we are copying the DX11VideoRenderer, possibly from here: https://github.com/microsoft/Windows-classic-samples/tree/main/Samples/DX11VideoRenderer ?
Do you think we could maybe build it into a lib and include .lib and .h files? or submoduling this directory could also be an option, I'm just wondering if we really need to expose(and maintain a separate copy) the source code for this renderer in our sample

Reply:
I've had to make some changes to the DX11VideoRenderer (you're correct that it's from the mentioned repo), so I don't think we can submodule the directory and immediately use it in the project.  The big one is that the sample uses C++14 and I've had to make some changes (eg. using typename modifiers to in templated classes) to get it compatible with C++17.